<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1959</width>
    <height>1098</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Hello</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>951</width>
          <height>1062</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="minimumSize">
            <size>
             <width>951</width>
             <height>531</height>
            </size>
           </property>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>220</y>
              <width>140</width>
              <height>140</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>140</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="minimumSize">
            <size>
             <width>933</width>
             <height>507</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QWidget" name="widget_9" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>271</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>191</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>300</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <widget class="QPushButton" name="pushButton_4">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>160</y>
                 <width>140</width>
                 <height>140</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>140</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>951</width>
          <height>1062</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="minimumSize">
            <size>
             <width>951</width>
             <height>531</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="widget_15" native="true">
              <widget class="QPushButton" name="pushButton_3">
               <property name="geometry">
                <rect>
                 <x>280</x>
                 <y>60</y>
                 <width>140</width>
                 <height>140</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>140</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="pushButton_5">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>100</y>
                 <width>140</width>
                 <height>140</height>
                </rect>
               </property>
               <property name="minimumSize">
                <size>
                 <width>140</width>
                 <height>140</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_14" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>140</width>
                   <height>140</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>766</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>933</width>
             <height>507</height>
            </size>
           </property>
           <widget class="QWidget" name="widget_13" native="true">
            <property name="geometry">
             <rect>
              <x>480</x>
              <y>120</y>
              <width>241</width>
              <height>251</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="widget_11" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QWidget" name="widget_10" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_6">
                     <property name="minimumSize">
                      <size>
                       <width>140</width>
                       <height>140</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>140</width>
                       <height>140</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>16</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>9</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_12" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_4"/>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
